configure.ac

```diff
@@ -1058,6 +1058,14 @@ if test "x$enable_afs" = "xyes"; then
 fi
 AM_CONDITIONAL([USE_AFSKRB], [grep HAVE_AFSKRB confdefs.h >/dev/null])
 
+AC_ARG_ENABLE(httppts,
+        [AS_HELP_STRING([--enable-httppts], [enable HTTP PTS support])],, [enable_httppts=no])
```

Comment:
Should we just default this on - or even require it?  It's annoying to add yet another compile time flag for a tiny bit of inoffensive code.

Reply:
I was thinking the same thing as I coded this up.  Initially I thought it might have to be gated due to using optional 3rd paty libraries, but that doesn't match reality.  I'll make the necessary changes.

Reply:
Great!  Fewer config options is always good.

Reply:
ptloader (and friends) is no built by default, with HTTP support